website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,441 @@
+---
+title: 'React Native 0.74 - Yoga 3.0, New Architecture Improvements, and more'
+authors: [hurali, alanjhughes, alfonsocj, huntie, cortinico]
+tags: [announcement, release, yoga]
+date: 2024-04-22
+---
+
+Today we're releasing React Native 0.74! This release adds Yoga 3.0, Bridgeless by default under the New Architecture, batched `onLayout` updates, and Yarn 3 as the default package manager for new projects.
+
+We are also actioning API removals for the deprecated `PropTypes` and `PushNotificationIOS` APIs. On Android, SDK 23 (Android 6.0) is now the minimum supported version.
+
+### Highlights
```

Comment:
Making these heading makes the ToC duplicated:

<img width="327" alt="Screenshot 2024-04-19 at 1 53 29 PM" src="https://github.com/facebook/react-native-website/assets/2440089/5cd7cd05-f51d-4b06-a4ed-722ef8496a38">

Reply:
It does (sadly) (note: still linkable as `#highlights-1`), which isn't perfect. I'm preserving this structure to match previous release posts, and for the sake of a structured outline.